ui/layout-pages-fixed.md

```diff
@@ -181,7 +181,7 @@ Additionally, the following settings are important:
 By default, this is switched off and therefore needs to be enabled.
 
 - **Embed SVG Flashing in Run-Mode**: It is possible within the editor to flash the elements of the SVG image that have been previously marked as interactive through the attribute 'openhab' (see below).
```

Comment:
"It is ... (see below)." in this line can be removed and only "This setting ... over them." be used.